common/constants.ts

```diff
@@ -155,18 +164,18 @@ export const WAZUH_DEFAULT_APP_CONFIG = {
   'ip.selector': true,
   'ip.ignore': [],
   'xpack.rbac.enabled': true,
-  'wazuh.monitoring.enabled': true,
-  'wazuh.monitoring.frequency': 900,
+  'wazuh.monitoring.enabled': WAZUH_MONITORING_DEFAULT_ENABLED,
+  'wazuh.monitoring.frequency': WAZUH_MONITORING_DEFAULT_FREQUENCY,
   'wazuh.monitoring.shards': WAZUH_INDEX_SHARDS,
   'wazuh.monitoring.replicas': WAZUH_INDEX_REPLICAS,
-  'wazuh.monitoring.creation': 'd',
+  'wazuh.monitoring.creation': WAZUH_MONITORING_DEFAULT_CREATION,
   'wazuh.monitoring.pattern': WAZUH_MONITORING_PATTERN,
-  'cron.prefix': 'wazuh',
-  'cron.statistics.status': true,
+  'cron.prefix': WAZUH_STATISTICS_DEFAULT_PREFIX,
+  'cron.statistics.status': WAZUH_STATISTICS_DEFAULT_STATUS,
   'cron.statistics.apis': [],
-  'cron.statistics.interval': '0 */5 * * * *',
-  'cron.statistics.index.name': 'statistics',
-  'cron.statistics.index.creation': 'w',
+  'cron.statistics.interval': WAZUH_STATISTICS_DEFAULT_CRON_FREQ,
```

Comment:
It seems the `cron.statistics.interval` changed from `0 */5 * * * *` to `0 * * * * *` (`WAZUH_STATISTICS_DEFAUL_CRON_FREQ=0 * * * * *`). Is this intentional?

Reply:
no its a bug